fails to build when async_kernel is not installed

```
#=== ERROR while installing graphql.~unknown ==================================#
# opam-version 1.2.2 (391297300524717e1475deaea37bca287c7f5e36)
# os           linux
# command      ocaml pkg/pkg.ml build --pinned true --with-lwt true --with-async false
# path         /home/ygrek/.opam/4.03.0/build/graphql.~unknown
# compiler     4.03.0
# exit-code    1
# env-file     /home/ygrek/.opam/4.03.0/build/graphql.~unknown/graphql-26798-d0f245.env
# stdout-file  /home/ygrek/.opam/4.03.0/build/graphql.~unknown/graphql-26798-d0f245.out
# stderr-file  /home/ygrek/.opam/4.03.0/build/graphql.~unknown/graphql-26798-d0f245.err
### stdout ###
# ocamlfind ocamldep -package angstrom,sexplib,ppx_sexp_conv,ppx_sexp_value,yojson,lwt,rresult,async_kernel -modules src/graphql.ml > src/graphql.ml.depends
# + ocamlfind ocamldep -package angstrom,sexplib,ppx_sexp_conv,ppx_sexp_value,yojson,lwt,rresult,async_kernel -modules src/graphql.ml > src/graphql.ml.depends
# ocamlfind: Package `async_kernel' not found
# Command exited with code 2.
### stderr ###
# pkg.ml: [ERROR] cmd ["ocamlbuild"; "-use-ocamlfind"; "-classic-display"; "-build-dir";
#      "_build"; "opam"; "pkg/META"; "CHANGES.md"; "LICENSE.md"; "README.md";
#      "src/graphql.a"; "src/graphql.cmxs"; "src/graphql.cmxa";
#      "src/graphql.cma"; "src/graphql_schema.cmx"; "src/graphql_schema.cmi";
#      "src/graphql_schema.mli"; "src/graphql_parser.cmx";
#      "src/graphql_parser.cmi"; "src/graphql_parser.mli"; "src/graphql.cmx";
#      "src/graphql.cmi"; "src/graphql.mli"; "src/graphql_lwt.a";
#      "src/graphql_lwt.cmxs"; "src/graphql_lwt.cmxa"; "src/graphql_lwt.cma";
#      "src/graphql_lwt.cmx"; "src/graphql_lwt.cmi"; "src/graphql_lwt.mli"]: exited with 10

```

--- a/_tags
+++ b/_tags
@@ -1,7 +1,9 @@
 true: bin_annot, debug, safe_string
 true: short_paths
 
-true: package(angstrom,sexplib,ppx_sexp_conv,ppx_sexp_value,yojson,lwt,rresult,async_kernel)
+true: package(angstrom,sexplib,ppx_sexp_conv,ppx_sexp_value,yojson,rresult)
+<src/*_async.*>: package(async_kernel)
+<src/*_lwt.*>: package(lwt)
 
 <src>: include
 <test>: include

--- a/pkg/META
+++ b/pkg/META
@@ -1,6 +1,6 @@
 description = "Library for writing GraphQL servers"
 version = "%%VERSION_NUM%%"
-requires = "angstrom sexplib ppx_sexp_conv ppx_sexp_value yojson cohttp.lwt rresult"
+requires = "angstrom sexplib ppx_sexp_conv ppx_sexp_value yojson rresult"
 archive(byte) = "graphql.cma"
 archive(native) = "graphql.cmxa"
 plugin(byte) = "graphql.cma"

